docs/source/customizations/overview.mdx

```diff
@@ -11,4 +11,142 @@ TODO:
 - Scripted plugins
 
 - Request lifecycle
-- Plugin trait 
\ No newline at end of file
+- Plugin trait
+
+---
+
+You can write custom plugins for Apollo Router to build functionality that isn't offered by default.  For example, make an external call to fetch some authentication data, or some complex logic.
+
+Custom plugins require that you build a router binary that includes your custom code. For this you should be generally familiar with building Rust projects, and have looked at the [examples provided in the router repo](https://github.com/apollographql/router/tree/main/examples).
+
+If you only need to make simple changes to a request or response then it's worth checking to see if [rhai](#rhai) script is enough to meet your needs. 
```

Comment:
Remove reference to rhai

Reply:
I don't disagree, but this PR is merged now.  I suspect this will get covered when someone eventually takes out a lock on the customizations overview page. 😉